File: frontend/src/Interfaces/Package.ts

export interface Package {
    packageId: string | number;
    packageName: string;
    packageDescription: string;
    packagePrice: number;
}

export interface CurrentPackage {
    currentPackageId: string | number;
    creatorId: string | number; // from table creator FK
    packageId: string | number; // from table package FK
    Date: string | Date; // ISO date
}

File: backend/backend/Entities/Package.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Entities
{
    public class Package
    {
        [Key]
        public int PackageID { get; set; } // Primary key
        public string PackageName { get; set; }
        public string PackageDescription { get; set; }
        public float PackagePrice { get; set; }
    }
}
